### ISSUE TITLE
Disable the global `ray()` function when not in debug mode

### ISSUE THREAD

**Issue by adamkiss** [genxbe/kirby3-ray#2]
Looking at the ray docs, you can disable the `ray()` call as well, with `ray()->disable()` - https://spatie.be/docs/ray/v1/usage/framework-agnostic-php-project#enabling--disabling-ray

I think it would be nice to disable that as well with this plugin. The only question is how exactly:

- we could add a hook after `system.loadPlugins:after`
- we could just append it to the `index.php` file, something like:

``` php
if (!option('debug') || !option('genxne.kirby-ray.enabled')){ ray()->disable(); }
```

perhaps even other way (put it to your custom class?)

**Comment by iskrisis** [comment]
`system.loadPlugins:after` would be much better than having special index.php?

**Comment by adamkiss** [comment]
@iskrisis You don't need special `index.php`, you can just add it after the `Kirby::plugin` in the main plugin PHP file:

``` php
<?php

Kirby::plugin(
// the current plugin
);

if (function_exists('ray') && (!option('debug') && !option('genxbe.kirby-ray.enabled'))) {
  ray()->disable();
}
```

**Comment by iskrisis** [comment]
I've deleted my comment so nobody can see proof of my stupidity ass brain in public.

**Comment by samzzi** [comment]
Thanks for your suggestions, fixed in 1.0.1. I hope I did everything right, these are my first adventures in managing a public package 😊
